packages/shopify-app-remix/src/server/authenticate/admin/billing/__tests__/check.test.ts

```diff
@@ -0,0 +1,203 @@
+import {
+  BillingInterval,
+  HttpResponseError,
+  SESSION_COOKIE_NAME,
+  Shopify,
+} from '@shopify/shopify-api';
+
+import {shopifyApp} from '../../../..';
+import {
+  APP_URL,
+  BASE64_HOST,
+  GRAPHQL_URL,
+  TEST_SHOP,
+  expectExitIframeRedirect,
+  getJwt,
+  getThrownResponse,
+  setUpValidSession,
+  signRequestCookie,
+  testConfig,
+  mockExternalRequest,
+} from '../../../../__test-helpers';
+import {REAUTH_URL_HEADER} from '../../../const';
+
+import * as responses from './mock-responses';
+
+const BILLING_CONFIG: Shopify['config']['billing'] = {
+  [responses.PLAN_1]: {
+    amount: 5,
+    currencyCode: 'USD',
+    interval: BillingInterval.Every30Days,
+  },
+};
+
+describe('Billing check', () => {
+  it('returns true when there is payment', async () => {
```

Comment:
Total nit, but we're not actually returning `true` here. Tiny but I went in expecting something else :)